api/query.go

```diff
@@ -13,13 +13,33 @@ import (
 	"github.com/equinor/vds-slice/internal/vds"
 )
 
-
+// Query for slice endpoints
+// @Description Query payload for slice endpoints /slice and /slice/metadata.
 type SliceQuery struct {
-	Vds       string  `form:"vds"       json:"vds"       binding:"required"`
-	Direction string  `form:"direction" json:"direction" binding:"required"`
-	Lineno    *int    `form:"lineno"    json:"lineno"    binding:"required"`
-	Sas       string  `form:"sas"       json:"sas"       binding:"required"`
-}
+	// The blob path to a vds on form: container/subpath
+	Vds string `form:"vds" json:"vds" binding:"required"`
+
+	// Direction can be specified in two domains
+	// - Annotation. Valid options: Inline, Crossline and Depth/Time/Sample
+	// - Index. Valid options: i, j and k.
+	//
+	// Only one of Depth, Time and Sample is valid for a given VDS. Which one
+	// depends on the depth units. E.g. Depth is a valid option for a VDS with
+	// depth unit "m" or "ft", but not if the units are "ms", "s".
+	// Sample is valid when the depth is unitless.
+	//
+	// i, j, k are zero-indexed and correspond to Inline, Crossline,
+	// Depth/Time/Sample, respectably.
+	//
+	// All options are case-insensitive.
+	Direction string `form:"direction" json:"direction" binding:"required"`
+
+	// Linenumber of the slice
+	Lineno *int `form:"lineno" json:"lineno" binding:"required"`
+
+	// A valid sas-token with read access to the container specified in Vds
```

Comment:
1. Maybe decide if it is vds, VDS or Vds?
2. Can we test it again on Radix? I am very confused. Sometimes it seems to me like container sas is enough. But if I now try Radix, not just I need to specify Iline instead of Inline, but also container sas doesn't fit, account one is needed.

Reply:
Something seems off with sas tokens or who knows what, so I will test it later, but in theory container should be enough.